### ISSUE TITLE
Maddie Deacon (26408664) - Code Review for Assignment  OOP

### ISSUE THREAD

**Issue by MaddieDeacon** [LMHedges/CMP1903M-A01-2223#3]
Documentation:

- Has created a start menu that allows users to have a choice of shuffle, deal or quit the program.
- Has used comments to contextualise his work therefore making it easier to understand and read through.
- Has created a class, called testing , which allows him to test individual sections of code or functions. therefore saving time through efficiency. 

Handling Errors: 
- has good use of error handling for integers out of range as well as different data types when not wanted however, when error message is printed it doesn't state where the error is, only that one exists.

in conclusion, the program is well documented and is clean as well as easy to understand. I would suggest to add more comments explaining some steps rather than just saying what the method does and for an error message to state the specific problem when found
